parlai/crowdsourcing/tasks/acute_eval/task_config/model_config_dataset.json

```diff
@@ -0,0 +1,10 @@
+{
+    "model1": {
```

Comment:
i know these are just examples, but we might want to make the key more relevant to the task --> e.g., have this be `"convai2_logs":` and `"ed_logs":`. i think these keys are what show up in analysis so it might be more clear there too

Reply:
Yeah, good point - changed in this case